meetings/2023-05-17.md

```diff
@@ -0,0 +1,72 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2023-05-17
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=lgl_Z2JjK_s>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1387>
+
+## Present
+
+* Colin Ihrig @cjihrig (voting member)
+* Danielle Adams @danielleadams (voting member)
+* Geoffrey Booth @GeoffreyBooth (voting member)
+* James Snell @jasnell (voting member)
+* Joyee Cheung @joyeecheung (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Robert Nagy @ronag (voting member)
+* Tobias Nießen @tniessen (voting member)
+
+## Agenda
+
+### Announcements
+
+* Michael, great collaborator summit/OpenJS World last week.
+* Ronag, blog post from Rafael on state of Node.js performance is a good read for everybody - <https://blog.rafaelgss.dev/state-of-nodejs-performance-2023>
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* No updates this week
+
+### nodejs/TSC
+
+* 2023 TSC Elections  [#1382](https://github.com/nodejs/TSC/issues/1382)
+  * On the agenda for awareness
+
+* Adding the OpenJS Foundation staff as a billing manager for nodejs-private
+[#1376](https://github.com/nodejs/TSC/issues/1376)
+  * Discussion ongoing in issue on how we might reduce usage
+  * No concerns/objections from the 9 TSC members who were in attendance
+  * Michael may want to try to avoid running automatically and instead kick them off manually in the private repo.
+  * Geoffrey, maybe run in order of increasing cost ?
+* Opportunity: Grace Hopper Open Source Day (virtual) Sept 22, 2023 [#1372](https://github.com/nodejs/TSC/issues/1372)
+  * Submission made, waiting to hear if accepted
```

Comment:
I cannot imagine they would turn us down. It would probably good to start getting organized now around who is going to curate the necessary tasks/issues.